mhs-reference-implementation/mhs/routing/tests/ldap_mocks.py

```diff
@@ -0,0 +1,33 @@
+from pathlib import Path
+
+from ldap3 import Server, Connection, MOCK_ASYNC
+import mhs.routing.sds as sds
+from definitions import ROOT_DIR
+
+MHS_TEST_DATA_PATH = Path(ROOT_DIR) / 'mhs' / 'routing' / 'tests' / 'data'
+SERVER_INFO_PATH = str(MHS_TEST_DATA_PATH / 'my_real_server_info.json')
+SCHEMA_PATH = str(MHS_TEST_DATA_PATH / 'my_real_server_schema.json')
+SERVER_ENTRIES_PATH = str(MHS_TEST_DATA_PATH / 'my_real_server_entries.json')
+
+
+def fake_ldap_server():
```

Comment:
This method name should probably be `fake_ldap_connection` now.

Reply:
Done